docs/usage_guide.rst

```diff
@@ -367,6 +367,15 @@ To stop tracking a particular account scroll down to the accounts tables and cli
 
 If an ethereum account also contains tracked tokens you can click on the arrow under "Actions" in order to expand its view and show the balance breakdown for the account over all assets it holds.
 
+For Bitcoin you can add addresses manually or let rotki discover them using a public key. From this key rotki knows how to generate your addresses and can query the Bitcoin's blockchain generating them until it finds an address without transactions.
+The different formats of allowed keys refer to how the addresses are generated, "P2PKH" uses keys starting with "xpub", "P2SH_P2WPKH" uses keys starting with "ypub" and "WPKH" uses keys starting with "zpub".
+This public keys also generate different types of addresses, "xpub"'s addresses have "1" as prefix, "ypub"'s addresses start with a "3" and addresses generated by "zpub" key start with "bc1".  When you paste your key in the "XPUB" field rotki will automatically detect the format.
```

Comment:
public keys -> xpub

Also instead of saying xpub, ypub, zpub here specify the type. P2PKH, P2SH_P2wPKH or WPKH.

Also as I explained above we don't always auto-detect the format. Only if the ypub/zpub is given. We can't otherwise. Can't assume that an xpub is always legacy (P2PKH).